app/assets/javascripts/admin/addon/controllers/admin-config-flags-settings.js

```diff
@@ -0,0 +1,12 @@
+import Controller from "@ember/controller";
+import { action } from "@ember/object";
+
+export default class AdminConfigFlagsSettingsController extends Controller {
+  filter = "";
+  queryParams = ["filter"];
```

Comment:
Not a big deal but I don't think you need this `queryParams`, I don't use it in the backups controller and it still links up properly

Reply:
Just tried and it stopped working. It works for you because you have it defined on route and I skipped route file completely. I decided to load setting directly in gjs component 

Backup route
```javascript
export default class AdminBackupsSettingsRoute extends Route {
  queryParams = {
    filter: { replace: true },
  };

  async model(params) {
    return {
      settings: await SiteSetting.findAll({ categories: ["backups"] }),
      initialFilter: params.filter,
    };
  }
}
```

Reply:
Ah yes right, makes sense :+1: